fluxc/src/main/java/org/wordpress/android/fluxc/network/rest/wpcom/dashboard/CardsRestClient.kt

```diff
@@ -163,6 +173,40 @@ class CardsRestClient @Inject constructor(
         )
     }
 
+    data class DynamicCardResponse(
+        @SerializedName("id") val id: String,
+        @SerializedName("title") val title: String,
+        @SerializedName("remote_feature_flag") val remoteFeatureFlag: String,
+        @SerializedName("featured_image") val featuredImage: String,
+        @SerializedName("url") val url: String,
+        @SerializedName("action") val action: String,
+        @SerializedName("order") val order: String,
+        @SerializedName("rows") val rows: List<DynamicCardRowResponse>
+    ) {
+        fun toDynamicCard() = DynamicCardModel(
+            id = id,
+            title = title,
+            remoteFeatureFlag = remoteFeatureFlag,
+            featuredImage = featuredImage,
+            url = url,
+            action = action,
+            order = CardOrder.fromString(order),
+            rows = rows.map { it.toDynamicCardRow() }
+        )
+    }
+
+    data class DynamicCardRowResponse(
+        @SerializedName("icon") val icon: String,
+        @SerializedName("title") val title: String,
+        @SerializedName("description") val description: String
```

Comment:
The same with nullability here. According to the designs we can have different modifications of rows: without an icon, title, or perhaps description. 
Reference: YSMw2nbFAgPpjZLLbYvTfT-fi-1281_9315
```suggestion
        @SerializedName("icon") val icon: String?,
        @SerializedName("title") val title: String?,
        @SerializedName("description") val description: String?
```

Reply:
As above https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2923#discussion_r1425181603